src/Stripe/Infrastructure/Requestor.cs

```diff
@@ -57,8 +83,10 @@ internal static WebRequest GetWebRequest(string url, string method, StripeReques
             if (requestOptions.IdempotencyKey != null)
                 request.Headers.Add("Idempotency-Key", requestOptions.IdempotencyKey);
 
-            request.ContentType = "application/x-www-form-urlencoded";
-            request.UserAgent = "Stripe.net (https://github.com/jaymedavis/stripe.net)";
+            request.Headers.UserAgent.Clear();
+            request.Headers.UserAgent.TryParseAdd($"Stripe.net {Version} (https://github.com/jaymedavis/stripe.net)");
```

Comment:
I like this change, but I need to run it by Stripe first. What was the thought process for removing the content type and clearing the user agent?